src/analytics/private/refresh-materialized-views-workflow.ts

```diff
@@ -0,0 +1,556 @@
+/**
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+import { join } from 'path';
+import { Duration } from 'aws-cdk-lib';
+import { ISecurityGroup, IVpc, SubnetSelection } from 'aws-cdk-lib/aws-ec2';
+import { IRole } from 'aws-cdk-lib/aws-iam';
+import { IFunction } from 'aws-cdk-lib/aws-lambda';
+import { RetentionDays, LogGroup } from 'aws-cdk-lib/aws-logs';
+import { StateMachine, LogLevel, IStateMachine, DefinitionBody, Wait, WaitTime, Succeed, Condition, Choice, Fail, TaskInput, Pass, Map, IntegrationPattern } from 'aws-cdk-lib/aws-stepfunctions';
+import { LambdaInvoke, StepFunctionsStartExecution } from 'aws-cdk-lib/aws-stepfunctions-tasks';
+import { Construct } from 'constructs';
+import { ExistingRedshiftServerlessCustomProps, ProvisionedRedshiftProps } from './model';
+import { createLambdaRole } from '../../common/lambda';
+import { createLogGroup } from '../../common/logs';
+import { REDSHIFT_MODE } from '../../common/model';
+import { SolutionNodejsFunction } from '../../private/function';
+import { WorkflowStatus } from '../private/constant';
+
+export interface RefreshMaterializedViewsWorkflowProps {
+  readonly appIds: string;
+  readonly projectId: string;
+  readonly securityGroupForLambda: ISecurityGroup;
+  readonly networkConfig: {
+    readonly vpc: IVpc;
+    readonly vpcSubnets: SubnetSelection;
+  };
+  readonly serverlessRedshift?: ExistingRedshiftServerlessCustomProps;
+  readonly provisionedRedshift?: ProvisionedRedshiftProps;
+  readonly databaseName: string;
+  readonly dataAPIRole: IRole;
+  readonly dataFreshnessInHour: number;
+  readonly timezoneWithAppId: string;
+}
+
+export class RefreshMaterializedViewsWorkflow extends Construct {
+  public readonly refreshMaterializedViewsMachine: IStateMachine;
+
+  private readonly lambdaRootPath = __dirname + '/../lambdas/refresh-materialized-views-workflow';
+  constructor(scope: Construct, id: string, props: RefreshMaterializedViewsWorkflowProps) {
+    super(scope, id);
+    this.refreshMaterializedViewsMachine = this.createWorkflow(props);
+  }
+
+  private createWorkflow(props: RefreshMaterializedViewsWorkflowProps): IStateMachine {
+    const logGroup = createLogGroup(this, { prefix: '/aws/vendedlogs/states/Clickstream/RefreshMaterializedViews' });
```

Comment:
Don't put lambda log in this prefix. Try to use the general prefix, like the default Lambda log.